### ISSUE TITLE
deprecated `request` dependency

### ISSUE THREAD

**Issue by DercilioFontes** [xoxco/node-slack#39]
I'm getting this warning:
``` shell
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
```

From my investigation, it is coming from `node-slack`

``` json
    "node-slack": {
      "version": "0.0.7",
      "resolved": "https://registry.npmjs.org/node-slack/-/node-slack-0.0.7.tgz",
      "integrity": "sha512-LsjUmymJcwF7P2Z3wImZRYIZvh068aQNYK6/FW+24fred4+hWPHyc6L9f7nQUoA6myt6NPFDBPXSqe493+cXqQ==",
      "requires": {
        "deferred": "0.7.1",
        "request": "~2.x"
      }
    },
```
Link: https://github.com/request/request/issues/3142
